catalog/resource_volumes.go

```diff
@@ -0,0 +1,83 @@
+package catalog
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type UpdateVolumeRequestContent struct {
+	// The name of the catalog where the schema and the volume are
+	CatalogName string `json:"catalog_name"`
+	// The comment attached to the volume
+	Comment string `json:"comment,omitempty"`
+	// The name of the schema where the volume is
+	SchemaName  string `json:"schema_name"`
+	FullNameArg string `json:"-" url:"-"`
+	// The name of the volume
+	Name string `json:"name"`
+	// The identifier of the user who owns the volume
+	Owner      string             `json:"owner,omitempty"`
+	VolumeType catalog.VolumeType `json:"volume_type"`
+	// The storage location on the cloud
+	StorageLocation string `json:"storage_location,omitempty"`
+}
+
+func ResourceVolumes() *schema.Resource {
+	// We cannot use catalog.UpdateVolumeRequestContent because it doesn't contain all the necessary fields
```

Comment:
Hi @nkvuong, @nfx suggested to use `catalog.UpdateVolumeRequestContent` but not sure how we can use that here since catalog_name and schema_name for example are required.